pandas/core/generic.py

```diff
@@ -7515,29 +7515,91 @@ def _check_percentile(self, q):
         return q
 
     _shared_docs['pct_change'] = """
-        Percent change over given number of periods.
+        Percentage change between the current and previous element.
+
+        This is useful in comparing the percentage of change in a time series
+        of elements.
 
         Parameters
         ----------
         periods : int, default 1
-            Periods to shift for forming percent change
+            Periods to shift for forming percent change.
         fill_method : str, default 'pad'
-            How to handle NAs before computing percent changes
+            How to handle NAs before computing percent changes.
         limit : int, default None
-            The number of consecutive NAs to fill before stopping
+            The number of consecutive NAs to fill before stopping.
         freq : DateOffset, timedelta, or offset alias string, optional
-            Increment to use from time series API (e.g. 'M' or BDay())
+            Increment to use from time series API (e.g. 'M' or BDay()).
+        kwargs : mapping, optional
+            A dictionary of keyword arguments passed into
```

Comment:
It's not a dictionary, but just additional keywords, so you cna make this ""Additional keyword arguments are passed .."

Reply:
But on second thought, are there actually additional keywords for shift?

Reply:
That's what it's mapped to https://github.com/pandas-dev/pandas/blob/7169830040255c1073f99e2ee8817596a67b1cd5/pandas/core/generic.py#L7561-L7562